modules/core/src/effects/lighting/camera-light.ts

```diff
@@ -8,15 +8,15 @@ export default class CameraLight extends PointLight {
     const {projectedLight} = this;
     const viewport = layer.context.viewport;
     const {coordinateSystem, coordinateOrigin, modelMatrix} = layer.props;
-    const {project_uCameraPosition} = getUniformsFromViewport({
+    const {cameraPosition} = getUniformsFromViewport({
       viewport,
       modelMatrix,
       coordinateSystem,
       coordinateOrigin
     });
     projectedLight.color = this.color;
     projectedLight.intensity = this.intensity;
-    projectedLight.position = project_uCameraPosition;
+    projectedLight.position = cameraPosition;
```

Comment:
Thought: I wonder if we should add a coordinate system suffix to position parameters. E.g. if this is the so called "common space":

```suggestion
    projectedLight.positionCS = cameraPositionCS;
```

Reply:
Perhaps, but it is out of scope for this PR - there are enough changes here as-is